resource_alks_iamrole.go

```diff
@@ -70,6 +111,41 @@ func resourceAlksIamRoleCreate(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+func resourceAlksIamTrustRoleCreate(d *schema.ResourceData, meta interface{}) error {
+	log.Printf("[INFO] ALKS IAM Trust Role Create")
+
+	var roleName = d.Get("name").(string)
+	var roleType = d.Get("type").(string)
+	var trustArn = d.Get("trust_arn").(string)
+
+	client := meta.(*alks.Client)
+
+	var resp *alks.IamRoleResponse
+	err := resource.Retry(2*time.Minute, func() *resource.RetryError {
```

Comment:
@brianantonelli Are you referring to this retry logic around the call to the client?

Reply:
Disregard my comment.. I expanded all and was looking at the imported client code.. 🤦‍♂️